semantics/language/common/expr/reference.k

```diff
@@ -67,6 +67,7 @@ module C-COMMON-EXPR-REFERENCE
      }*/
      rule *(tv(Loc:SymLoc, T:PointerType))
           => checkRestrict(Loc)
+          ~> checkLoc(Loc)
```

Comment:
Yeah, I suppose ids for static duration stuff will be replaced by their addresses during translation. It might be better to tone that down a bit.